src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent, useRef, useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ApplicationStart } from 'src/core/public';
+import { createNavigateToUrlClickHandler } from './click_handler';
+
+export interface RedirectCrossAppLinksProps extends React.HTMLAttributes<HTMLDivElement> {
+  navigateToUrl: ApplicationStart['navigateToUrl'];
+  currentAppId$: ApplicationStart['currentAppId$'];
```

Comment:
Just curious but does this mean the API for this component is changing too? The old one just required the whole `ApplicationStart` as `application`. I mainly ask about this because if this is an intended change, we really need to start documenting it somewhere. There's other `.mdx` docs that reference the old version of this explicitly citing the use of `application`. Also, the comments below now seem to be out of date as well.

Reply:
Good call - I can document this in the mdx file at the bare minimum 👍🏻

Reply:
[f11aa17](https://github.com/elastic/kibana/pull/124197/commits/f11aa17f3f0f14a06e6ed56d62c481326d20a2c0)